### PR TITLE
Menu Create for presentation

### PR DESCRIPTION
**Pull request by stped18**
this is menu for the presentaten 
TO DO after presentation 
add screens to models

### CODE REVIEW

File: dk.sdu.se4.core/src/main/java/dk/sdu/se4/core/StartMenu.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+                } catch (InterruptedException e) {
+                    e.printStackTrace();
+                }
+                logger.debug("Sound play {}", this.exitsound);
+                Gdx.app.exit();
+            }
+        }else{
+            this.game.batch.draw(this.exitActivebtn,(float)(this.game.width/2)-(this.exitActivebtn.getWidth()/2),(float) (this.game.height/3));
+        }
+        if(Gdx.input.getX()< (this.game.width/2.66)+this.playbtn.getWidth()&& Gdx.input.getX()>(this.game.width/2.66) &&
```
Comment (Kjalris) [inline dk.sdu.se4.core/src/main/java/dk/sdu/se4/core/StartMenu.java:92]: Can this be rewritten to be more readable? Or maybe some comments?

File: dk.sdu.se4.core/src/main/resources/META-INF/gamescreen.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+               name="PluginService" policy="static" unbind="removePlugin"/>
+   
+    
+    <reference bind="addProcessorService" cardinality="0..n"
+               interface="dk.sdu.se4.common.service.ProcessorService"
+               name="ProcessorService" policy="static" unbind="removeProcessorService" />
+    
+    <reference bind="addPostProcessorService" cardinality="0..n"
+               interface="dk.sdu.se4.common.service.PostProcessorService"
+               name="PostProcessorService" policy="static" unbind="removePostProcessorService" />-->
```
Comment (Nicklason) [inline dk.sdu.se4.core/src/main/resources/META-INF/gamescreen.xml:24]: remove comments

**Review by Kjalris — COMMENTED** [review]
Before i approve i would like some comments / more readable code (where i left the comment)

**Review by Nicklason — CHANGES_REQUESTED** [review]
Look through your what you are commiting before doing it. It is a waste of time to look through many small changes that doesn't do anything.

**Review by DenBedste — APPROVED** [review]

**Pull request closed without merging**